life_server/Functions/MySQL/fn_insertRequest.sqf

```diff
@@ -7,24 +7,21 @@
     Adds a player to the database upon first joining of the server.
     Recieves information from core\sesison\fn_insertPlayerInfo.sqf
 */
-private ["_queryResult","_query","_alias"];
 params [
-    "_uid",
-    "_name",
+    ["_uid","",[""]],
+    ["_name","",[""]],
     ["_money",-1,[0]],
-    ["_bank",-1,[0]],
-    ["_returnToSender",objNull,[objNull]]
```

Comment:
Did you remove the passing of the player object from the client file here?

Reply:
Sorted.